xo-install.sh

```diff
@@ -628,6 +628,11 @@ function PrepInstall {
                     runcmd "rm -rf $INSTALLDIR/xo-builds/xen-orchestra-$TIME"
                     exit 0
                     ;;
+                *)
+                    printinfo "Cleaning up install directory: $INSTALLDIR/xo-builds/xen-orchestra-$TIME"
+                    runcmd "rm -rf $INSTALLDIR/xo-builds/xen-orchestra-$TIME"
+                    exit 0
+                    ;;
```

Comment:
I'd simply replace the `n)` option with the `*)` to get the same outcome and avoid repeating same lines. `y` is the only answer which should proceed to update so everything else can quit. Could also remove the default value from line 621 as it won't be needed.

Reply:
I'd thought of that, but then it differs from every other instance of the same case logic in the script :)

Reply:
True, there's couple of similar ones. It's fine though, i can take care of unifying those with this after.

Reply:
Updated in latest commit.